### PR TITLE
Feat contact

### PR DESCRIPTION
**Pull request by austinmargarone**
I just have 2 errors that i'm not sure how to resolve.

### CODE REVIEW

File: components/contact/ContactForm.tsx [hunk 1]
```diff
@@ -24,10 +24,11 @@ export default function ContactForm() {
 
     emailjs
       .sendForm(
-        process.env.SERVICE_ID,
-        process.env.TEMPLATE_ID,
+        "service_l76u8u8",
+        "template_msyezf2",
         form.current,
```
Comment (brandonetter) [inline components/contact/ContactForm.tsx]:
```suggestion
        form.current!,
```

Tells typescript this won't be a null value

File: components/contact/ContactForm.tsx
Comment (brandonetter) [inline components/contact/ContactForm.tsx:1]: Also let's edit

```js
const form = useRef();
```

to
```js
const form = useRef(null);
```

And that should fix your other error.

**Review by brandonetter — CHANGES_REQUESTED** [review]
Not sure what the exact errors your facing are- but here are two typescript ones that you can fix.

Looks like emailjs sends back a 'success' but I don't get an email.

Is it working for you?

**Comment by austinmargarone** [comment]
@brandonetter 
Yup, just those two typescript errors. 
Yup, I get an email. Should I have one sent to the client as well? I plan on implementing toasts eventually. 
I'm having issues with my dotenv when I try to process that way it doesn't work and I always seem to have an issue with my git ignore on the .env file. I've tried to purge it from GitHub but am having issues.

**Comment by brandonetter** [comment]
If your .env keeps ending up in the git tracking, then you probably accidentally created it and ran 'git add' before adding it to your .gitignore.

You can remove it from the 'cache' in git so it will no longer be tracked with this:
```js
git rm -r --cached .env
```

**Comment by brandonetter** [comment]
Also for your variable naming:

Take a look at these docs:
https://nextjs.org/docs/app/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser

For the EmailJS public keys you should prefix them with `NEXT_PUBLIC` so that NextJS knows to bundle them with the client's browser code.

**Comment by austinmargarone** [comment]
@brandonetter Awesome, thanks.

**Pull request merged**